generator/templates/Default/src/router.js

```diff
@@ -8,77 +11,80 @@ export default new Router({
     base: process.env.BASE_URL,
     routes: [
         {
-            path: '/',
-            name: 'home',
-            component: () => import('@/views/Home'),
-        },
-        {
-            path: '/403',
-            name: '403',
-            component: () => import('./views/PageForbidden.vue'),
+            path: '',
+            redirect: {name: 'home'},
         },
         <%_ if (options.useAuthorisation) { _%>
         {
-            path: '/login',
-            name: 'login',
-            component: () => import('@/views/Login'),
-            <%_ if (options.useTemplateLoader) { _%>
-            meta: {
-                template: () => import('./templates/Authorisation.vue'),
-            },
-            <%_ } _%>
-        },
-        {
-            path: '/password/forgotten',
-            name: 'password.forgotten',
-            component: () => import('./views/PasswordForgotten.vue'),
-            <%_ if (options.useTemplateLoader) { _%>
-            meta: {
-                template: () => import('./templates/Authorisation.vue'),
-            },
-            <%_ } _%>
+            path: '',
+            component: () => import('@/templates/Authorisation'),
+            children: [
+                {
+                    path: '/login',
+                    name: 'login',
+                    component: () => import('@/views/Login'),
+                },
+                {
+                    path: '/password/forgotten',
+                    name: 'password.forgotten',
+                    component: () => import('./views/PasswordForgotten.vue'),
+                },
+                {
+                    path: '/password/reset/:token',
+                    name: 'password.reset',
+                    component: () => import('./views/PasswordReset.vue'),
+                },
+                {
+                    path: '/invitation/accept/:token',
+                    name: 'invitation.accept',
+                    component: () => import('./views/InvitationAccept.vue'),
+                },
+            ]
         },
+         <%_ } _%>
         {
-            path: '/password/reset/:token',
-            name: 'password.reset',
-            component: () => import('./views/PasswordReset.vue'),
-            <%_ if (options.useTemplateLoader) { _%>
-            meta: {
-                template: () => import('./templates/Authorisation.vue'),
-            },
+            path: '',
+            <%_ if (options.useAuthorisation) { _%>
+            beforeEnter: AuthorisationGuard,
```

Comment:
There's still this type across the 'Authorisation' thing. Should be 'Authorization'

Reply:
Authorisationis english With a z is american
I could change it, but not in this PR that's a change everywhere beginning with the promts which means this would be a breaking change

Reply:
It's called Authorization in mobuyou? xd